context/src/junit/validator.rs

```diff
@@ -65,6 +65,17 @@ pub fn validate(report: &Report) -> JunitReportValidation {
             }
         };
 
+        if let Some(raw_test_suite_id) = test_suite.extra.get("id") {
+            let test_case_id = uuid::Uuid::parse_str(raw_test_suite_id).unwrap_or_default();
+            if test_case_id.get_version() != Some(uuid::Version::Sha1) {
+                test_suite_validation.add_issue(JunitValidationIssue::Invalid(
+                    JunitTestSuiteValidationIssueInvalid::TestSuiteInvalidId(
+                        raw_test_suite_id.to_string().clone(),
+                    ),
+                ));
+            }
+        }
+
```

Comment:
@dfrankland I added some validation + tests. Let me know if you want to review this before I merge.